qiskit/providers/ibmq/runtime/__init__.py

```diff
@@ -102,11 +102,12 @@
     qc.measure_all()
 
     # Execute the circuit using the "circuit-runner" program.
-    program_inputs = {'circuits': circuit, 'measurement_error_mitigation': True}
+    param = provider.runtime.program(program_id="circuit-runner").parameter
+    param.param1 = 123
     options = {'backend_name': backend.name()}
     job = provider.runtime.run(program_id="circuit-runner",
-                               options=options,
-                               inputs=program_inputs)
+                            options=options,
+                            inputs=params)
```

Comment:
You probably want to test the examples to make sure they still work (let me know if you don't have access). In this case `params` is not defined, and `param1` is not a valid parameter to `circuit-runner`.

Reply:
Fixed!